frontend/packages/operator-lifecycle-manager/src/components/modals/edit-registry-poll-interval-modal.tsx

```diff
@@ -67,8 +64,8 @@ const EditRegistryPollIntervalModal: React.FC<EditRegistryPollIntervalModalProps
         <ModalTitle>{t('olm~Edit registry poll interval')}</ModalTitle>
         <ModalBody>
           <FormGroup label={t('olm~Registry poll interval')} fieldId="pollInterval_dropdown">
-            <Dropdown
-              className="dropdown--full-width"
+            <ConsoleSelect
```

Comment:
update registry poll interval doesn't work

https://github.com/user-attachments/assets/02f9ad65-06af-42b7-bf6b-f971c2506e67

Reply:
this also looks broken on `4.20.0-0.ci-2025-07-23-021242`, can we resolve this in another PR?

Reply:
resolved in #15337